src/pages/designing/get-started.mdx

```diff
@@ -13,85 +13,100 @@ running.
+IBMers should get a license for Figma, our primary design kit tool. You may also
+access tools we no longer support, Sketch, Adobe XD, and Axure, by heading to
+the [Design Toolbox](https://w3.ibm.com/design/toolbox/). However, we no longer
+maintain or update the kits for these tools. We recommend you migrate to Figma
+to get the most updated kits we offer.
 
 <Row className="resource-card-group">
+
 <Column colLg={4} colMd={4} noGutterSm>
   <ResourceCard
-    subTitle="IBM Design Language"
-    href="https://www.ibm.com/design/language/"
-    >
-
-![Bee icon](/images/bee.svg)
-
+    subTitle="Get Figma"
+    href="https://w3.ibm.com/design/toolbox/?&_ga=2.49792614.1500760159.1682969477-2020417520.1680805833#/ui-design-tools/figma/README"
+    actionIcon="launch">
+    <MdxIcon name="figma" />
   </ResourceCard>
-
 </Column>
-<Column colLg={4} colMd={4} noGutterSm>
-  <ResourceCard
-    subTitle="IBM Brand Systems"
-    href="https://www.ibm.com/brand/systems"
-    >
+</Row>
 
-![Bee icon](images/bee.svg)
+## Step 3: Get the Figma design kits
+
+We maintain the core Carbon kits in [Figma](/designing/kits/figma). These kits
+include all the Carbon core components and styles for reuse in your product and
+web experiences. By using the kits you will automatically receive updates made
+to the Carbon libraries, ensuring your designs stay up-to-date with the latest
+releases. This allows you to speak the same language and have shared
+functionality expectations as your developers in the hand-off and development
+cycle.
+
+Head on over to the [Design kits](/designing/kits/figma) section and follow the
+instructions to get set up with Figma.
+
+### Core design kits
+
+Everyone using Carbon should use the following Figma core design kits as a
+starting point for building user interfaces. This brings consistency and speed,
+allowing teams to focus on solving more specific user tasks. Core design kits
+include elements, core components, wireframes, and guidelines.
+
+| Maintainer    | Design kit                                                                                                                     | Type         |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------ | ------------ |
+| IBM Brand     | [IBM Pictograms](https://www.figma.com/file/PkUl9UBuvA41GPpyl84NBc/Pictograms---IBM-Design-Language)                           | `Elements`   |
+|               | [IBM Icons](https://www.figma.com/file/J5c0d85dSJn9JnBhSYYLmD/Icons---IBM-Design-Language?node-id=129%3A2)                     | `Elements`   |
+|               | [v11 Text styles](<https://www.figma.com/file/rK06GY6bvEfokuzrFGdtWB/(v11)-Text-Styles---IBM-Design-Language?node-id=129%3A2>) | `Elements`   |
+|               | [v11 Color styles](https://www.figma.com/file/Gvwx2RnAZzDKTjVuC5xOlO/Color-Styles---IBM-Design-Language?node-id=129%3A2)       | `Elements`   |
```

Comment:
This isn't a v11 library; this is just the general color swatches like Green 10 - Green 100.  The v11 color styles that are our tokens actually live inside the theme files.
```suggestion
|               | [IBM Color palette](https://www.figma.com/file/Gvwx2RnAZzDKTjVuC5xOlO/Color-Styles---IBM-Design-Language?node-id=129%3A2)       | `Elements`   |
```